Classes/Flowpack/Neos/FrontendLogin/Controller/AuthenticationController.php

```diff
@@ -6,15 +6,35 @@
  *                                                                             */
 
 use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Error\Error;
 use TYPO3\Flow\Mvc\ActionRequest;
 use TYPO3\Flow\Security\Authentication\Controller\AbstractAuthenticationController;
+use TYPO3\Flow\Security\Exception\AuthenticationRequiredException;
 
 /**
  * Controller for displaying a login/logout form and authenticating/logging out "frontend users"
  */
 class AuthenticationController extends AbstractAuthenticationController {
 
 	/**
+	 * @var \TYPO3\Flow\I18n\Translator
+	 * @Flow\Inject
+	 */
+	protected $translator;
+
+	/**
+	 * @Flow\InjectConfiguration(package="Flowpack.Neos.FrontendLogin", path="translation.packageKey")
+	 * @var string
+	 */
+	protected $translationPackageKey;
+
+	/**
+	 * @Flow\InjectConfiguration(package="Flowpack.Neos.FrontendLogin", path="translation.sourceName")
+	 * @var string
+	 */
+	protected $translationSourceName;
```

Comment:
I don't think that we need this much flexibility, assuming the default (= `Main`) here is perfectly fine IMO